extensions/modules/file/src/test/xquery/modules/file/read-binary.xqm

```diff
@@ -19,19 +19,35 @@
  : License along with this library; if not, write to the Free Software
  : Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  :)
-xquery version "3.0";
+xquery version "3.1";
 
-module namespace read-binary="http://exist-db.org/testsuite/modules/file/read-binary";
+module namespace readbinary="http://exist-db.org/testsuite/modules/file/read-binary";
+import module namespace helper="http://exist-db.org/xquery/test/util/helper" at "resource:util/helper.xqm";
+import module namespace fixtures="http://exist-db.org/xquery/test/util/fixtures" at "resource:util/fixtures.xqm";
 
 import module namespace test="http://exist-db.org/xquery/xqsuite" at "resource:org/exist/xquery/lib/xqsuite/xqsuite.xql";
```

Comment:
I think you can remove the `at "resource:org/exist/xquery/lib/xqsuite/xqsuite.xql";` bit as that is an old legacy way of doing this import for the test suite.